### PR TITLE
VZ-10077.  Update thanos Object Storage example for 1.6

### PR DESCRIPTION
**Pull request by michael-cico**
Fix up the `storage.yaml` example in the Thanos doc

### CODE REVIEW

File: content/en/docs/observability/monitoring/configure/thanos.md [hunk 1]
```diff
@@ -55,13 +55,13 @@ spec:
...
 
-## Enable long-term storage
+## Enable long-term storage using OCI Object Storage
 
 Optionally, to enable long-term storage of metrics complete the following steps:
 
 ### Step 1: Create a YAML configuration file
 
-Create a `storage.yaml` file using OCI object storage.
+Create a local file named `storage.yaml` with valid credentials for accessing your bucket in OCI Object Storage.
```
Comment (desagar) [inline content/en/docs/observability/monitoring/configure/thanos.md:64]: maybe we should also say that they should use their region and bucket name in addition to their credentials.

**Review by desagar — APPROVED** [review]

File: content/en/docs/observability/monitoring/configure/thanos.md [hunk 2]
```diff
@@ -55,13 +55,13 @@ spec:
 </div>
 {{< /clipboard >}}
 
-## Enable long-term storage
+## Enable long-term storage using OCI Object Storage
```
Comment (rosemarymarano) [inline content/en/docs/observability/monitoring/configure/thanos.md:58]: -> using OCI object storage (using sentence capitalization, I don't think that object storage needs to be capitalized unless it's a product name)

File: content/en/docs/observability/monitoring/configure/thanos.md (hunk 1 above)
Comment (rosemarymarano) [inline content/en/docs/observability/monitoring/configure/thanos.md:64]: -> OCI object storage. (Again, I don't think this needs to be capitalized unless it's a product name).

**Review by rosemarymarano — APPROVED** [review]
edits or leave as is, according to whether OCI object storage is a product name.

File: content/en/docs/observability/monitoring/configure/thanos.md (hunk 2 above)
Comment (michael-cico) [reply content/en/docs/observability/monitoring/configure/thanos.md:58]: Yes, it's a product name, I looked at this before using the caps:

https://docs.oracle.com/en-us/iaas/Content/Object/Concepts/objectstorageoverview.htm

File: content/en/docs/observability/monitoring/configure/thanos.md (hunk 1 above)
Comment (michael-cico) [reply content/en/docs/observability/monitoring/configure/thanos.md:64]: It is, see other comments

File: content/en/docs/observability/monitoring/configure/thanos.md [hunk 3]
```diff
@@ -55,13 +55,17 @@ spec:
...
 
-Optionally, to enable long-term storage of metrics complete the following steps:
+Optionally, Thanos can be configured to use [OCI Object Storage](https://docs.oracle.com/en-us/iaas/Content/Object/Concepts/objectstorageoverview.htm)
```
Comment (rosemarymarano) [inline content/en/docs/observability/monitoring/configure/thanos.md:60]: -> Optionally, you can configure Thanos to use...

**Review by rosemarymarano — COMMENTED** [review]
One minor tweak to your changed text.

File: content/en/docs/observability/monitoring/configure/thanos.md (hunk 1 above)
Comment (michael-cico) [reply content/en/docs/observability/monitoring/configure/thanos.md]: fixed

File: content/en/docs/observability/monitoring/configure/thanos.md (hunk 3 above)
Comment (michael-cico) [reply content/en/docs/observability/monitoring/configure/thanos.md]: Fixed

**Pull request merged**